search/serializers.py

```diff
@@ -86,6 +87,48 @@ def postprocess_fields(self, discussions_obj, serialized_data):
         return {"author_channel_membership": sorted(get_channels(discussions_obj.user))}
 
 
+class ESCourseSerializer(ESSerializer):
```

Comment:
Could we possibly have this be a direct model serializer instead like `PostSerializer` and `CommentSerializer` are now?

Reply:
Switched to `ESProxySerializer` like `ESProfileSerializer`